helm/helm.go

```diff
@@ -81,6 +81,10 @@ func main() {
 					Name:  "force, aye-aye",
 					Usage: "Perform install even if dependencies are unsatisfied.",
 				},
+				cli.BoolFlag{
+					Name:  "dry-run",
+					Usage: "Run but do not install into Kubernetes.",
```

Comment:
Run vs install is confusing verbiage to me... Maybe something like: "show but do not execute kubectl commands"?